phpunit.xml

```diff
@@ -2,7 +2,7 @@
 <phpunit
```

Comment:
Just one last last thing, can you please also add `phpunit.xml` to `.gitattributes` `export-ignore`?

Reply:
Done